index.html

```diff
@@ -0,0 +1,250 @@
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+                <img class="img" src="image/card-2.svg" alt="" />
+            </div>
+
+            <div class="card">
+                <img class="img" src="image/card-3.svg" alt="" />
+                <div class="content">
+                    <div class="topic">Tonic</div>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <div class="topic">
+                        Multi-Post
+                        <br />
+                        Stories
+                    </div>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+                <img class="img" src="image/card-4.svg" alt="" />
+            </div>
+        </div>
+    </div>
+
+    <div id="about" class="myself">
```

Comment:
- [ ] Kindly add a `background-image or color` to the **_` About me`_** section to match the Figma design template chosen  👍🏼. Your  **_` About me`_** section is currently missing a background.  Kindly add it 👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/185760836-efdfb158-9503-4baa-8241-831adca1010d.png) |![image](https://user-images.githubusercontent.com/63473584/185760871-263d0ee1-cc20-471f-b45c-f0d2502f8053.png)  |

Please add padding to the container section as well and the correct spacing to the elements as specified in the Figma design template 👍🏼